shared/modules/consensus_module.go

```diff
@@ -30,3 +31,26 @@ type ConsensusModule interface {
 	CurrentStep() uint64
 	ValidatorMap() ValidatorMap
 }
+
+// This interface represents functions exposed by consensus module to be access by pacemaker module.
+// This functions should be only called by the pacemaker module.
+// TODO This interface will be removed when asynchronous interaction among pacemaer and consensus is built.
+// See issue for details: https://github.com/pokt-network/pocket/issues/428
+type ConsensusPacemaker interface {
+	//Pacemaker Consensus interaction modules
+	ClearLeaderMessagesPool()
```

Comment:
Should this just be rename to "ResetRound" ?

Reply:
Makes sense.